pkgs/by-name/in/insync/package.nix

```
@@ -7,38 +7,26 @@
 , dpkg
 , nss
 , alsa-lib
-, libvorbis
-, libdrm
-, libGL
-, wayland
 , xkeyboard_config
 , libthai
 , libsForQt5
 }:
 
 let
   pname = "insync";
-  # Find a binary from https://www.insynchq.com/downloads/linux#ubuntu.
-  version = "3.8.7.50516";
-  ubuntu-dist = "mantic_amd64";
-  insyncDeb = (fetchurl {
-    urls = [
-      "https://cdn.insynchq.com/builds/linux/insync_${version}-${ubuntu-dist}.deb"
-      "https://web.archive.org/web/20240409080945/https://cdn.insynchq.com/builds/linux/insync_${version}-${ubuntu-dist}.deb"
-    ];
-    hash = "sha256-U7BcgghbdR7r9WiZpEOka+BzXwnxrzL6p4imGESuB/k=";
-  });
-  insyncEmblemIconsDeb = (fetchurl {
-    urls = [
-      "https://cdn.insynchq.com/builds/linux/insync-emblem-icons_${version}_all.deb"
-      "https://web.archive.org/web/20240409081214/https://cdn.insynchq.com/builds/linux/insync-emblem-icons_${version}_all.deb"
-    ];
-    hash = "sha256-uALaIxETEEkjDTx331uIsb4VswWk2K0dGuDMYH8v5U8=";
-  });
+  # Find a binary from https://www.insynchq.com/downloads/linux
+  version = "3.9.0.60000";
+  ubuntu-dist = "trixie_amd64";
   insync-pkg = stdenvNoCC.mkDerivation {
     name = "${pname}-pkg-${version}";
```

```suggestion
    pname = "${pname}-pkg";
    inherit version;
```
we should properly set pname+version

---

done